### ISSUE TITLE
Javascript error and wrong upload path

### ISSUE THREAD

**Issue by rubenheymans** [linchpinstudios/yii2-filemanager#1]
When I want to upload an image, I get this error in firebug:
Error: Syntax error, unrecognized expression: /yii2-app-advanced/backend/web/filemanager/files/#
throw new Error( "Syntax error, unrecognized expression: " + msg );

Also the image do not upload, and after uploading I get the error in firebug:
"NetworkError: 404 Not Found -x/backend/web/filemanager/files/files/thumbnails/1411041436-550e95258718ddd46707505efe45c06e.jpg"

**Comment by wisdomroma** [comment]
Do you have issue? I have this error too(((

**Comment by PendalF89** [comment]
Yes, I have had this error. But now don't checked. I've already written my [file manager](https://github.com/PendalF89/yii2-filemanager) ;)

**Comment by linchpinstudios** [comment]
Sorry Pendal for the delay. Your system looks nice.

I've added a directory to the Module so that it looks for the directory. It seems to not be doing that on your configurations.

Let me know if the latest update works for you.

If you want to move it to the @frontend of your advanced configuration you can set that in your config file. Also note you should set the url for your front-end domain. 

```
    'modules' => [
        'filemanager' => [
            'direcotry' => '@frontend/',
            'url' => 'http://frontend.com/'
        ],
```

**Comment by PendalF89** [comment]
Thank you!

**Comment by QuinLena15** [comment]
I have this error too(( What to do?

**Comment by ivantree** [comment]
me to
